internal/services/costmanagement/export_resource_base.go

```diff
@@ -373,10 +404,16 @@ func flattenExportDefinition(input *exports.ExportDefinition) []interface{} {
 		queryType = string(input.Type)
 	}
 
+	version := ""
+	if input.DataSet != nil && input.DataSet.Configuration != nil && input.DataSet.Configuration.DataVersion != nil {
+		version = *input.DataSet.Configuration.DataVersion
```

Comment:
Can make use of `pointer.From` here again 
```suggestion
	if input.DataSet != nil && input.DataSet.Configuration != nil {
		version = pointer.From(input.DataSet.Configuration.DataVersion)
```